buildah.go

```diff
@@ -28,7 +28,7 @@ const (
 	Package = "buildah"
 	// Version for the Package.  Bump version in contrib/rpm/buildah.spec
 	// too.
-	Version = "1.19.8"
+	Version = "1.19.9-dev"
```

Comment:
I don't think this PR is necessary.  ON the supported branches we don't have -dev packages.

Reply:
Dropped the specific commit :+1: